charts/tempo-distributed/templates/compactor/poddisruptionbudget-compactor.yaml

```diff
@@ -2,12 +2,12 @@
 apiVersion: {{ include "tempo.pdb.apiVersion" . }}
 kind: PodDisruptionBudget
 metadata:
-  name: {{ include "tempo.compactorFullname" . }}
+  name: {{ include "tempo.resourceName" (dict "ctx" . "component" "compactor") }}
```

Comment:
what's the rule of thumb for when to extract a `$dict` variable vs not?

Reply:
Good question.  I tend to think it gets easier to read the more we do it.  I think this was brought in by a rebase and I just missed it.  I'll update to use a `$dict`.